akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -289,13 +289,14 @@ final private[stream] class QueueSink[T]() extends GraphStageWithMaterializedVal
   type Requested[E] = Promise[Option[E]]
 
   val in = Inlet[T]("queueSink.in")
+  override def initialAttributes = DefaultAttributes.queueSink
```

Comment:
since this is here now we can also remove the `withAttributes` from the implementation of `Sink.queue`